### ISSUE TITLE
Navigation

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2242]
Navigation lists should be in `<nav>` wrappers, and since we have several, they should be labeled.

Also, the though there are aria attributes in the main navbar and the dropdown inside of it, they aren't working as intended. See if adjusting to the markup recommended by https://getbootstrap.com/docs/3.3/components/#navbar takes care of it. We might also need to upgrade to a more recent boostrap js.

Finally: try to put the navbar's dropdown and its contents in context. Landing on a button with your name on it isn't going to suggest the riches inside; there are elements inside that appear to be headings of sorts, but aren't marked up as such, etc.
